app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (1, 0, [0, 0]),
+        (0, 1, [0, 0]),
+        (15, 0, [1, 0]),
+        (0, 15, [0, 1]),
+        (24, 0, [2, 0]),
+        (1000, 1000, [246, 197]),
+        (10000, 10000, [2496, 1997]),
+        (-1, 15, [0, 1]),
+        (15, -1, [1, 0]),
+        (-1, -1, [0, 0]),
+    ]
+)
+def test_get_human_age(
+    cat_age: int,
+    dog_age: int,
+    expected: get_human_age
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
```

Comment:
add test for this case:
if the function receives an incorrect type of data it raises the correct exception.